### DISCUSSION TITLE
Next Release?

### DISCUSSION THREAD

**Discussion by aguelen-fhit** [stephanstapel/ZUGFeRD-csharp#286] [Q&A]
Hi there,
I'm now using the ZUGFeRD-charp release version 12.8.0. But currently I need to set "DespatchAdviceReferencedDocument" (BT-16). I have seen that you have already added it to the project. And the question is, is there a new release plan? when will you release it?

Best regards.
Akif

**Comment by stephanstapel** [comment]
Hi @aguelen-fhit ,
there are a lot of breaking changes in the library. Thus I am a bit reluctant to write the readme. If you could use the code from the repository and test it thoroughly, that'd be of great help for me to then release a new nuget package.
